File: Exams/Retake-Exam-8-January-2018/Java/src/main/java/trainsystem/entity/Trip.java
package trainsystem.entity;

import javax.persistence.*;

@Entity
@Table(name = "trips")
public class Trip {
    private Integer id;

    private String origin;

    private String destination;

    private int business;

    private int economy;

    public Trip(Integer id, String origin, String destination, int business, int economy) {
        this.id = id;
        this.origin = origin;
        this.destination = destination;
        this.business = business;
        this.economy = economy;
    }

    public Trip() {
        String origin = this.getOrigin();
        String destination = this.getDestination();
        int business = this.getBusiness();
        int economy = this.getEconomy();
    }

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    @Column(name = "origin", nullable = false)
    public String getOrigin() {
        return origin;
    }

    public void setOrigin(String origin) {
        this.origin = origin;
    }

    @Column(name = "destination", nullable = false)
    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    @Column(name = "business", nullable = false)
    public int getBusiness() {
        return business;
    }

    public void setBusiness(int business) {
        this.business = business;
    }

    @Column(name = "economy", nullable = false)
    public int getEconomy() {
        return economy;
    }

    public void setEconomy(int economy) {
        this.economy = economy;
    }
}

File: Exams/Retake-Exam-8-January-2018/PHP/src/AppBundle/Entity/Trip.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Trip
 *
 * @ORM\Table(name="trips")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\TripRepository")
 */
class Trip
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="origin", type="string", length=255)
     */
    private $origin;

    /**
     * @var string
     *
     * @ORM\Column(name="destination", type="string", length=255)
     */
    private $destination;

    /**
     * @var int
     *
     * @ORM\Column(name="business", type="integer")
     */
    private $business;

    /**
     * @var int
     *
     * @ORM\Column(name="economy", type="integer")
     */
    private $economy;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set origin
     *
     * @param string $origin
     *
     * @return Trip
     */
    public function setOrigin($origin)
    {
        $this->origin = $origin;

        return $this;
    }

    /**
     * Get origin
     *
     * @return string
     */
    public function getOrigin()
    {
        return $this->origin;
    }

    /**
     * Set destination
     *
     * @param string $destination
     *
     * @return Trip
     */
    public function setDestination($destination)
    {
        $this->destination = $destination;

        return $this;
    }

    /**
     * Get destination
     *
     * @return string
     */
    public function getDestination()
    {
        return $this->destination;
    }

    /**
     * Set business
     *
     * @param integer $business
     *
     * @return Trip
     */
    public function setBusiness($business)
    {
        $this->business = $business;

        return $this;
    }

    /**
     * Get business
     *
     * @return int
     */
    public function getBusiness()
    {
        return $this->business;
    }

    /**
     * Set economy
     *
     * @param integer $economy
     *
     * @return Trip
     */
    public function setEconomy($economy)
    {
        $this->economy = $economy;

        return $this;
    }

    /**
     * Get economy
     *
     * @return int
     */
    public function getEconomy()
    {
        return $this->economy;
    }
}

